modules/reactor/src/positions/HexagonalGridPositions.C

```diff
@@ -58,9 +58,20 @@ HexagonalGridPositions::HexagonalGridPositions(const InputParameters & parameter
         std::set<unsigned int>(getParam<std::vector<unsigned int>>("include_in_pattern").begin(),
                                getParam<std::vector<unsigned int>>("include_in_pattern").end()))
 {
-  if (MooseUtils::absoluteFuzzyGreaterThan(_nr * _pin_pitch * sqrt(3), _lattice_flat_to_flat))
-    paramError("lattice_flat_to_flat",
-               "Bundle pitch is too small to fit this many rings at that pin pitch");
+  if (_nr == 1)
+  {
+    if (MooseUtils::absoluteFuzzyGreaterThan(_pin_pitch, _lattice_flat_to_flat))
+      paramError("lattice_flat_to_flat",
+                 "For one ring, the lattice flat to flat must be at least the pin pitch");
+  }
+  else
+  {
+    if (MooseUtils::absoluteFuzzyGreaterThan((_nr - 1) * _pin_pitch * sqrt(3),
+                                             _lattice_flat_to_flat))
```

Comment:
so from your testing, specifying a flat to flat that is smaller than `nr * pitch * sqrt(3)`  breaks the utility right?
so until the utility is reworked to allow half pins, it should error when the flat to flat is below that number

I know the utility still works for cases with 5.5 pins but only with specifying:
nr = 6
flat to flat. = flat to flat from 6 rings (not the current flat to flat of 5.5 pins)